src/pages/developers/tutorials/localnet.mdx

```diff
@@ -0,0 +1,173 @@
+Localnet is a local development environment that simplifies the development of
+universal apps.
+
+Localnet:
+
+- Starts an [Anvil](https://book.getfoundry.sh/anvil/) local testnet node
+- Deploys [protocol
+  contracts](https://github.com/zeta-chain/protocol-contracts/tree/main/v2) on
+  the local testnet node. Both EVM gateway and ZetaChain gateway are deployed
```

Comment:
Can these link to explanations of what `EVM gateway`, and `ZetaChain gateway` are and how they differ? It's not clear if I start at the tutorial.

Reply:
Good point! Will add a link from this tutorial to https://www.zetachain.com/docs/developers/evm/gateway/

Reply:
bd2e165